src/main/java/jenkins/branch/MultiBranchProjectViewHolder.java

```diff
@@ -223,7 +223,7 @@ public ViewImpl(ViewGroup owner, @NonNull SCMHeadCategory category) {
          */
         @Override
         public String getDisplayName() {
-            return category.getDisplayName().toString();
+            return category.getDisplayName().toString() + " (" + getItems().size() + ")";
```

Comment:
Is there any significant cost to calling `getItems()` here, particularly for a tab we are not currently displaying?